PyFunceble/cli/system/launcher.py

```diff
@@ -676,6 +676,14 @@ def generate_percentage_file(parent_dirname: str) -> None:
             ):
                 break
 
+            generate_percentage_file(protocol["destination"])
```

Comment:
This one change shouldn't be there. What the hell is happening here ?

Reply:
This is not the changes I made.... https://mypdns.org/spirillen/PyFunceble/-/commit/a2ff7278884be2c6cbda05cb3d753ef505611446